test/groups/general_unitary_groups.jl

```diff
@@ -1,261 +0,0 @@
-            @test isapprox(Un, p, X, X2)
-            q2 = exp_lie(Un, X)
-            X3 = log_lie(Un, q2)
-            @test isapprox(Un, p, X, X3)
-            @test inv(Un, p) == adjoint(p)
-            @test injectivity_radius(Un, p) == π
-        end
-    end
-
-    @testset "Quaternionic Unitary Group" begin
-        QU1 = Unitary(1, ℍ)
-        @test repr(QU1) == "Unitary(1, ℍ)"
-
-        @test identity_element(QU1) === Quaternion(1.0)
-
-        p = QuaternionF64(
-            0.4815296357756736,
-            0.6041613272484806,
-            -0.2322369798903669,
-            0.5909181717450419,
-        )
-        X = Quaternion(0.0, 0, 0, 1)
-        q = exp(QU1, p, X)
-        X2 = log(QU1, p, q)
-        @test isapprox(QU1, p, X, X2)
-        q2 = exp_lie(QU1, X)
-        X3 = log_lie(QU1, q2)
-        @test isapprox(QU1, p, X, X3)
-        q3 = Manifolds.exp_fused(QU1, p, X, 1.0)
-        @test isapprox(QU1, q, q3)
-
-        q3 = Ref(Quaternion(0.0))
-        exp_lie!(QU1, q3, X)
-        @test isapprox(QU1, q3[], q2)
-        X4 = Ref(Quaternion(0.0))
-        log_lie!(QU1, X4, q2)
-        @test isapprox(QU1, identity_element(QU1), X3, X4[])
-
-        X5 = fill(Quaternion(0.0), 1, 1)
-        log_lie!(QU1, X5, fill(q2, 1, 1))
-        @test isapprox(QU1, identity_element(QU1), X3, X5[])
-
-        @test inv(QU1, p) == conj(p)
-
-        @test project(QU1, p, Quaternion(1.0, 2.0, 3.0, 4.0)) ===
-            Quaternion(0.0, 2.0, 3.0, 4.0)
-    end
-
-    @testset "Special Unitary Group" begin
-        SU2 = SpecialUnitary(2)
-        @test repr(SU2) == "SpecialUnitary(2)"
-
-        p = ones(2, 2)
-        q = project(SU2, p)
-        @test is_point(SU2, q; error = :error)
-        q2 = allocate(q)
-        project!(SU2, q2, p)
-        @test q == q2
-        p2 = copy(p)
-        p2[1, 1] = -1
-        q2 = project(SU2, p2)
-        @test is_point(SU2, q2; error = :error)
-        p3 = [2.0 0; 0.0 2.0] #real pos determinant
-        @test project(SU2, p3) == p3 ./ 2
-        Xe = ones(2, 2)
-        X = project(SU2, q, Xe)
-        @test is_vector(SU2, q, X)
-        @test_throws DomainError is_vector(SU2, p, X, true; error = :error) # base point wrong
-        @test_throws DomainError is_vector(SU2, q, Xe, true; error = :error) # Xe not skew hermitian
-        @test_throws DomainError is_vector(
-            SU2,
-            Identity(AdditionOperation()),
-            Xe,
-            true;
-            error = :error,
-        ) # base point wrong
-        e = Identity(MultiplicationOperation())
-        @test_throws DomainError is_vector(SU2, e, Xe, true; error = :error) # Xe not skew hermitian
-
-        @test manifold_volume(SpecialUnitary(1)) ≈ 1
-        @test manifold_volume(SpecialUnitary(2)) ≈ 2 * π^2
-        @test manifold_volume(SpecialUnitary(3)) ≈ sqrt(3) * π^5
-        @test manifold_volume(SpecialUnitary(4)) ≈ sqrt(2) * 4 * π^9 / 12
-    end
-
-    @testset "SO(4) and O(4) exp/log edge cases" begin
```

Comment:
Should we keep these tests in Manifolds.jl as tests for `exp`/`log` or move them to LieGroups.jl?

Reply:
I think they are already done as edge cases on Liegroups.jl? See e.g. https://github.com/JuliaManifolds/LieGroups.jl/blob/main/test/LieGroupsTestSuite.jl/so4_edge_cases.jl where they are defined even within the test suite.

Reply:
Maybe you are right and added them back already for the matrices. So we can close this.